spec/features/movies/index_spec.rb

```diff
@@ -65,7 +64,6 @@
         click_button 'Search'
 
         expect(current_path).to eq(movies_path)
-        expect(page).to have_content('0 Movies')
```

Comment:
I removed the 'movie count' header from the movies index as the wireframe doesn't call for this info. It was leftover from our initial buildout of the view.